coalib/output/printers/LogPrinter.py

```diff
@@ -1,5 +1,5 @@
-import traceback
 import logging
+import traceback
```

Comment:
Some of the other projects are still using it (coala-bears & coala-quickstart use it among the ones I know, there can be more). So, the plan is to file the issues for all the projects which uses it, and then close all of them first, and at last, remove it. I've already filed issue to remove it at https://github.com/coala/coala/issues/4484. Please mark it as blocked at the moment. I'll file issues & try to remove it from all the other projects, and then at last remove it..